README.md

```diff
@@ -8,31 +8,7 @@ Alternatively, you can set two separate environment variables GITHUB_TOKEN_SOURC
 
 ## Usage
 
-This supports different options:
-
-```yml
-only_sync_on_label:
-  description: 'If set, will only sync on issues with a label of this text'
-  required: false
-repo_source:
-  description: 'Org/Repo slug for the source repository. Will default to action launch repo if not set.'
-  required: false
-repo_target:
-  description: 'Org/repo slug for the target repository.'
-  required: true
-only_sync_main_issue:
-  description: 'Will exclude the syncing of comments.'
-  required: false
-  default: "false"
-additional_issue_labels:
-  description: 'Additional labels to add to the target issues on sync (comma-separated)'
-  required: false
-  default: ""
-sync_repo_labels:
-  description: 'Whether to sync the labels from source to target repository'
-  required: false
-  default: "true"
-```
+See the list of supported options in `action.yml`
  
 ## Example
```

Comment:
not sure whether it makes sense to adjust the example to include the edited event for comments:
```
  issue_comment:
    types: [created, edited]
```

or to add a comment that one should enable it when run with `only_sync_main_issue: false`

Reply:
looking at the code, `deleted` as well.

Reply:
Good idea, added the two options there.